server/db/db.js

```diff
@@ -636,6 +637,49 @@ async function getInstallationWithSessionId(sessionId, clientParam) {
   return null
 }
 
+async function getHistoricAlertsByAlertApiKey(alertApiKey, maxHistoricAlerts, maxTimeAgoInMillis, clientParam) {
+  let client = clientParam
+  const transactionMode = typeof client !== 'undefined'
+
+  try {
+    if (!transactionMode) {
+      client = await pool.connect()
+    }
+
+    // Historic Alerts are those with status "Completed" or that were last updated longer ago than the SESSION_RESET_TIMEOUT
+    const { rows } = await client.query(
+      `
+      SELECT s.id, b.unit, s.incident_type, s.num_presses, s.created_at, s.responded_at
+      FROM sessions AS s
+      LEFT JOIN buttons AS b ON s.button_id = b.button_id
+      LEFT JOIN installations AS i ON s.installation_id = i.id
+      WHERE i.alert_api_key = $1
+      AND (
+        s.state = $2
+        OR s.updated_at < now() - $3 * INTERVAL '1 millisecond'
+      )
+      ORDER BY s.created_at DESC
+      LIMIT $4
+      `,
```

Comment:
What do you two think of this for multi-line string. It looks much more readable to me.

Reply:
this is much better!

Reply:
also the comment makes it easier to understand what the query is doing. thanks.